src/sync/incomingSyncUtils.js

```diff
@@ -230,8 +233,11 @@ export function createOrUpdateRecord(database, settings, recordType, record) {
       const thisStoreId = settings.get(THIS_STORE_ID);
       const sequenceKey = SEQUENCE_KEYS.translate(record.name, EXTERNAL_TO_INTERNAL, thisStoreId);
       // Don't accept updates to number sequences
-      if (database.objects('NumberSequence')
-                  .filtered('sequenceKey == $0', sequenceKey).length > 0) break;
+      if (
+        database.objects('NumberSequence').filtered('sequenceKey == $0', sequenceKey).length > 0
```

Comment:
Subjective haha. The bonus with prettier is that if we don't debate it, we don't spend time debating/implementing about how exactly to format something best.